### DISCUSSION TITLE
请问设计是否可以支持Antd5x，如果已经支持可以说一下具体步骤吗，多谢

### DISCUSSION THREAD

**Discussion by SeedHuang** [alibaba/designable#400] [Q&A]
你好，我们现在项目想要使用这个设计器，但是有个问题，目前我们项目是antd5.x的，但是designable支持的4.x，如果不降级到antd4.x就会有很多less样式的问题，请问是否已经支持Antd5x，或者是否能够告诉怎么配置可以适应5.x，非常感谢

**Comment by bacium** [comment]
请问你在拉下来启动项目的时候有没有报错，我的提示是这个：
“ERROR in /Users/itars/Desktop/code/formily/designable/node_modules/@monaco-editor/react/dist/index.mjs 1:1806
Module parse failed: Unexpected token (1:1806)“
我不知道怎么改了，node版本是16.19.1
